mirdata/track.py

```diff
@@ -33,11 +33,78 @@ def __repr__(self):
             if val.__doc__ is None:
                 raise ValueError("{} has no documentation".format(prop))
 
-            val_type_str = val.__doc__.split(':')[0]
+            val_type_str = val.__doc__.split(":")[0]
             repr_str += "  {}: {},\n".format(prop, val_type_str)
 
         repr_str += ")"
         return repr_str
 
     def to_jams(self):
         raise NotImplementedError
+
+
+class MultiTrack(Track):
```

Comment:
just a quick question: 
I am interested in score-informed source separation. in this case, each track and target should have annotations (scores).
so in my case, I should extend this the Track and MultiTrack objects with score-related methods, similarly to other mirdatasets loaders which deal with their particular annotations?

Reply:
yes exactly!